src/popup/components/dashboard/Card.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div
+    v-if="!closable || (!hiddenCards || !hiddenCards.includes(closable))"
```

Comment:
Bracket isn't needed
```suggestion
    v-if="!closable || !hiddenCards || !hiddenCards.includes(closable)"
```